.github/workflows/avm.platform.toggle-avm-workflows.yml

```diff
@@ -0,0 +1,48 @@
+name: 'avm.platform.toggle-avm-workflows'
+
+on:
+  workflow_dispatch:
+    inputs:
+      workMode:
+        type: choice
+        description: "Enable or disable workflows"
+        required: true
+        options:
+          - "enable"
+          - "disable"
+        default: "disable"
+      includePattern:
+        type: string
+        description: "RegEx which workflows are included"
+        required: false
+        default: "avm.res"
+      excludePattern:
```

Comment:
General question. Why not only a pattern? Once can filter to a specific set just with a single regex

Reply:
That was asked in our call, to also have an exclusion pattern

Reply:
Ah well, maybe there is a use case somewhere even though I don't see it right now :D Will resolve